superset/dashboards/api.py

```diff
@@ -954,6 +955,11 @@ def import_(self) -> Response:
                     overwrite:
                       description: overwrite existing dashboards?
                       type: boolean
+                    configOverwrite:
+                      description: determine which fields to overwrite
+                      type: Dict<str, bool>
+                      example: {'dashboards': True, 'charts': False ...}
```

Comment:
Nit, the payload is JSON, not Python:

```suggestion
                      example: {"dashboards": true, "charts": false ...}
```